candi/version_map.yml

```diff
@@ -60,6 +60,29 @@ bashible: &bashible
         gcp:
           desiredVersion: "5.4.0-1029-gcp"
           allowedPattern: ""
+    '22.04':
+      docker:
+        desiredVersion: "docker-ce=5:20.10.14~3-0~ubuntu-jammy"
+        allowedPattern: ""
+        containerd:
+          desiredVersion: "containerd.io=1.5.11-1"
+          allowedPattern: "containerd.io=1.[56]"
+      containerd:
+        desiredVersion: "containerd.io=1.5.11-1"
+        allowedPattern: "containerd.io=1.[56]"
+      kernel:
+        generic:
+          desiredVersion: "5.15.0-27-generic"
+          allowedPattern: ""
```

Comment:
Why not to add `allowedPattern: "5.15"` to allow minor patches to kernel? @nabokihms What do you think?